openapi/stellar-disbursement-platform/main.yml

```diff
@@ -2166,6 +2168,14 @@ definitions:
         description: 'Enables multi-user approval workflow for disbursements'
         type: boolean
         example: true
+      sms_registration_message_template:
+        description: The custom SMS message template set for the registration invite. The default message isn't returned.
+        type: string
+        example: Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+      otp_message_template:
+        description: The custom OTP message template set to send OTP code to the receiver complete the registration flow. The default message isn't returned.
+        type: string
+        example: Lorem ipsum dolor sit amet, consectetur adipiscing elit. The default message isn't returned.
```

Comment:
@CaioTeixeira95 can you delete 'The default message isn't returned' from the example?

I'm happy to write real examples if helpful.

Reply:
Thank you, @torisamples! Removed `The default message isn't returned`.

Reply:
Feel free to provide any real-world examples.

Reply:
Hey @CaioTeixeira95 I was talking to Tori about this and we should probably add real examples here as it's not obvious or intuitive how we can tokenize the messages. 

```
"sms_registration_message_template": "You have a payment waiting for you from {{.OrganizationName}}. Click {{.RegistrationLink}} to register.",

"otp_message_template": "{{.OTP}} is your {{.OrganizationName}} phone verification code."
```